test/format/ops.c

```diff
@@ -1395,6 +1400,23 @@ ops(void *arg)
             break;
         }
 
+        if (mirrored_truncate) {
```

Comment:
It'd be nice to check `op == TRUNCATE && table->mirror` here, but we null out `table` on line 1309 and there's some `goto`s in between that line and here that make it hard to understand the impact of moving the `table = tinfo->table = NULL;` call after this new logic. To keep things simple I've added the mirrored_truncate